ReactNativeClient/lib/shim-init-node.js

```diff
@@ -412,6 +419,24 @@ function shimInit() {
 	shim.pathRelativeToCwd = (path) => {
 		return toRelative(process.cwd(), path);
 	};
+
+	shim.isKeytarAvailable = () => {
```

Comment:
As this is only used by your code, please remove it from the shim and keep as a private helper method.

Reply:
By private helper method, you mean a separate file -- something like `keytar-utils.js` in which the function will reside. Am I right?

Reply:
No, it can be a simple function in shim-init-node. Just `function isKeytarAvailable() { .... }`

Reply:
Oh okay. Thanks!